skylark_library.bzl

```diff
@@ -1,109 +1,7 @@
-# Copyright 2017 The Bazel Authors. All rights reserved.
-#
-# Licensed under the Apache License, Version 2.0 (the "License");
-# you may not use this file except in compliance with the License.
-# You may obtain a copy of the License at
-#
-#    http://www.apache.org/licenses/LICENSE-2.0
-#
-# Unless required by applicable law or agreed to in writing, software
-# distributed under the License is distributed on an "AS IS" BASIS,
-# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-# See the License for the specific language governing permissions and
-# limitations under the License.
+load("//:bzl_library.bzl", "StarlarkLibraryInfo", "bzl_library")
 
```

Comment:
Should we add a `print()` to nudge people into migrating?  Like -
  https://github.com/bazelbuild/bazel-skylib/blob/1099dd2d0a80649700e3dff0cfc759cfe0f86c6e/lib.bzl#L17-L21

Reply:
Sounds good to me. Done.